warehouse/query-core/src/main/java/datawave/query/tld/TLD.java

```diff
@@ -43,11 +92,18 @@ public static ByteSequence parseFieldAndValueFromFI(ByteSequence cf, ByteSequenc
         return new ArrayByteSequence(fnFv);
     }
     
+    /**
+     * Parses the FIELD and VALUE from a Term Frequency key.
+     *
+     * @param cq
+     *            - the term frequency key's ColumnQualifier
+     * @return - a byte sequence that is the field and value separated by a null byte
+     */
     public static ByteSequence parseFieldAndValueFromTF(ByteSequence cq) {
-        ArrayList<Integer> deezNulls = instancesOf(0, cq, 2);
-        ArrayList<Integer> deezOtherNulls = lastInstancesOf(0, cq, 1);
-        final int startFn = deezOtherNulls.get(0) + 1, stopFn = cq.length();
-        final int startFv = deezNulls.get(1) + 1, stopFv = deezOtherNulls.get(0);
+        ArrayList<Integer> nulls = instancesOf(0, cq, 2);
+        ArrayList<Integer> otherNulls = lastInstancesOf(0, cq, 1);
+        final int startFn = otherNulls.get(0) + 1, stopFn = cq.length();
```

Comment:
Not your typo, but I feel like this was intended to be `startFv` for "field/value"

Reply:
Ah, should've read the next line. Nevermind.